### PR TITLE
Initial training of the model

### PR DESCRIPTION
**Pull request by ShaeBrown**
Had to round to the nearest second for onset detection :crying_cat_face: 
Also added some of the data to the test_data folder
Results:

Onset detection captured 489 out of 705 training data events
Using cross validation with  10 folds
Label ranking average precision:  0.498191525535
Coverage error 18.4915540541
Ranking loss 0.450886780744
Class                  Accuracy  Precision    Recall
-------------------  ----------  -----------  --------
Bass drum                 0.748  0.748        1.000
Bongo                     1      n/a          n/a
Chinese ride cymbal       0.99   0.000        0.000
Clave                     0.98   0.000        n/a
Cowbell                   0.941  0.000        0.000
Crash cymbal              0.743  0.053        0.007
Crash cymbal 2            0.948  0.000        0.000
Cross stick               0.894  0.333        0.086
Hi-hat closed             0.576  0.475        0.113
Hi-hat open               0.797  0.000        0.000
Low tom                   0.948  0.053        0.071
Lowest tom                0.995  0.000        n/a
Mid tom                   0.944  0.000        0.000
Ride cymbal               0.938  0.000        0.000
Shaker                    0.985  0.000        n/a
Snare drum                0.606  0.611        0.938
Snare drum brush          1      n/a          n/a
Snare rim shot            0.965  0.000        0.000
Splash cymbal             0.976  0.000        0.000
Tambourine                0.938  0.083        0.037
Timbala                   0.965  0.000        0.000
Timbala rim shot          0.966  0.053        0.333

### CODE REVIEW

File: train.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    export = input().lower().strip() == "y"
+    if export:
+        print("Enter file name")
+        file = input()
+        joblib.dump(clf, './trained_models/' + file + '.pkl')
+        print("Exported model " + file + " in trained models")
+
+
+if __name__ == '__main__':
+    train()
```
Comment (jarredhawkins) [inline train.py]: you forgot a blank newline at the end of file -- going to have to reject this pr due to low code quality

**Comment by jarredhawkins** [comment]
Those results are awesome!!

I haven't got around to actually trying it on my local machine but I trust ya, merge whenever!

File: train.py (hunk 1 above)
Comment (ShaeBrown) [reply train.py]: shit, wouldn't blame you if you dumped me as a project partner too..

**Comment by ShaeBrown** [comment]
The results for bass drum are really good, but there is low recall for some of the other classes. Some of them are really obscure, maybe we could group similar sounding ones together. :woman_shrugging: 

I'll merge it, and i'll push the rest of the training data straight to master. Idk if we are violating licenses by uploading the training data but i just want to make sure we have the same folder structure

**Comment by jarredhawkins** [comment]
Yeah I wouldnt worry about the license until/if we make this repo public.
 
Maybe the recall could be something to do with the onset detection sensitivity... I can play around with it when I manage to scrape up some free time

**Comment by ShaeBrown** [comment]
I think the repo is public, but I can make it private for now. Sounds good, if you can work on increasing the accuracy, I can work on hooking it up to the front end sometime.

**Pull request merged**